paper/paper.md

```diff
@@ -31,122 +31,109 @@ affiliations:
+    + `WeaklyCmpressibleSPHSystem` (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+    + `EntropicallyDampedSPHSystem` (EDAC): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
 
 * *Structure Systems*
-    + [`TotalLagrangianSPHSystem` (TLSPH)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/total_lagrangian_sph/): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
+    + `TotalLagrangianSPHSystem` (TLSPH): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
     + `DEMSystem`: Discrete element system [@Bicanic:2004], [@Cundall:1979].
 
 * *Boundary Systems*
-    + [`BoundarySPHSystem`](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/boundary/) with several boundary models where each model follows a different interaction rule.
-    + `OpenBoundarySPHSystem`: System to simulate non-reflecting boundary conditions [@Lastiwka:2009]
+    + `BoundarySPHSystem` with several boundary models where each model follows a different interaction rule.
+    + `OpenBoundarySPHSystem`: System to simulate non-reflecting (open) boundary conditions [@Lastiwka:2009]
 
-The code presented here can be found on [GitHub](https://github.com/trixi-framework/TrixiParticles.jl),  along with a detailed [manual](https://trixi-framework.github.io/TrixiParticles.jl/stable/) explaining how to use the package. Additionally, we provide tutorials explaining how to setup a simulation for fluids, structure mechanics or a fluid-structure interaction. All implemented examples for testing are located in the [`examples/`](https://github.com/trixi-framework/TrixiParticles.jl/tree/main/examples) folder, thus, the results can be easily reproduced.
-
-### Feature highlights
-
-As a young project that aims to be the Julia code for particle-based simulation we expect to integrate other methods such as incompressible SPH (ISPH) [@Liu:2022] or heat conduction with SPH [@Biriukov:2018] and couple them among each other and even with mesh-based methods.
-
-TODO: Instead of listing it in the paper, it might be better to link the [feature list](https://trixi-framework.github.io/TrixiParticles.jl/stable/#Features)
-So far, the following feature highlights have been implemented:
+* *Correction methods and models*
+  + Density diffusion [@Antuono:2010]
+  + Transport-velocity formulation (TVF) [@Adami:2013]
+  + Intra-particle-force surface tension [@Akinci:2013]
+  + Kernel corrections [@Akinci:2013], [@Bonet:1999], [@Basa:2009]
```

Comment:
We're not actively using these, as it doesn't yield the results that we would like to see. I suggest we don't list them here.

Reply:
Yes.